assemblies/pentaho-war/pom.xml

```diff
@@ -22,10 +22,10 @@
     <hibernate-ehcache.version>3.6.9.Final</hibernate-ehcache.version>
     <commons-io.version>2.2</commons-io.version>
     <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
-    <tika-core.version>1.19.1</tika-core.version>
+    <tika-core.version>1.22</tika-core.version>
     <xmlgraphics-commons.version>2.2</xmlgraphics-commons.version>
     <c3p0.version>0.9.1.2</c3p0.version>
-    <jackrabbit.version>2.14.2</jackrabbit.version>
+    <jackrabbit.version>2.16.5</jackrabbit.version>
```

Comment:
Please move the version property to the parent pom repository and use dependency management.